### PR TITLE
Repeat cluster.initial_master_nodes log warning

### PR DESCRIPTION
**Pull request by DaveCTurner**
Today we log a warning about removing `cluster.initial_master_nodes` at startup, but this is easy to miss in amongst all the other messages. With this commit we repeat the warning message every 12 hours.

Relates #92741

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/cluster/coordination/ClusterBootstrapService.java [hunk 1]
```diff
@@ -129,14 +129,9 @@ void logBootstrapState(Metadata metadata) {
             if (bootstrapRequirements.isEmpty()) {
                 logger.info("this node is locked into cluster UUID [{}] and will not attempt further cluster bootstrapping", clusterUUID);
```
Comment (idegtiarenko) [inline server/src/main/java/org/elasticsearch/cluster/coordination/ClusterBootstrapService.java:130]: NIT: might call `logRemovalWarning` as well

File: test/framework/src/main/java/org/elasticsearch/test/MockLogAppender.java [hunk 2]
```diff
@@ -208,4 +213,18 @@ private static String getLoggerName(String name) {
         }
         return COMMON_PREFIX + name;
     }
+
+    public Releasable capturing(Class<?>... classes) {
```
Comment (tlrx) [inline test/framework/src/main/java/org/elasticsearch/test/MockLogAppender.java:217]: Nice

**Review by tlrx — APPROVED** [review]
LGTM

**Review by idegtiarenko — APPROVED** [review]

**Comment by DaveCTurner** [comment]
@elasticmachine update branch

**Comment by DaveCTurner** [comment]
@elasticmachine please run elasticsearch-ci/part-1

**Pull request merged**